### PR TITLE
Added WebAuthn4J module

### PR DESCRIPTION
**Pull request by FroMage**
Almost the same API as the WebAuthn module, but backed by WebAuthn4J

Related to https://github.com/vert-x3/vertx-dependencies/pull/197 and an upcoming vertx-web PR

### CODE REVIEW

File: vertx-auth-webauthn4j/src/main/java/module-info.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+module io.vertx.auth.webauthn4j {
+
+  requires transitive io.vertx.auth.common;
```
Comment (vietj) [inline vertx-auth-webauthn4j/src/main/java/module-info.java]: for now I am trying to avoid transitive requires in Vert.x 5

**Comment by vietj** [comment]
in vertx5, MANIFEST.MF files are not used anymore since we use module declaration, you can remove it

File: vertx-auth-webauthn4j/src/main/java/module-info.java (hunk 1 above)
Comment (FroMage) [reply vertx-auth-webauthn4j/src/main/java/module-info.java]: Removed

**Comment by FroMage** [comment]
Removed manifest.

File: vertx-auth-webauthn4j/src/main/asciidoc/enums.adoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
+= Enums
```
Comment (vietj) [inline vertx-auth-webauthn4j/src/main/asciidoc/enums.adoc]: I think we can remove this file

**Comment by vietj** [comment]
Minor comment, LGTM, good work Steph

**Pull request merged**